lib/osf-components/addon/components/contributors/card/readonly/template.hbs

```diff
@@ -34,5 +38,46 @@
         >
             {{t (concat 'osf-components.contributors.citation.' @contributor.bibliographic)}}
         </span>
+        <span
+            data-test-contributor-caret={{@contributor.id}}
+            data-analytics-name={{
+                if this.showDropdown
+                'Expand employment and education info'
+                'Collapse employment and education info'
+            }}
+            local-class='CardSection'
+            role='button'
+            {{on 'click' this.toggleDropdown}}
+        >
+            <FaIcon
+                @icon={{if this.showDropdown 'caret-up' 'caret-down'}}
+            />
+        </span>
     </div>
+    {{#if this.showDropdown}}
+        <div data-test-contributor-card-dropdown local-class='DropdownSection'>
+            <ul local-class='DropdownList'>
+                {{#if (or this.employmentList this.educationList)}}
+                    {{#each this.employmentList as |item|}}
+                        <li>
+                            {{#if (and item.institution item.department)}}
+                                {{concat item.institution ', ' item.department}}
+                            {{else}}
+                                {{item.institution}}
+                            {{/if}}
+                        </li>
+                    {{/each}}
+                    {{#each this.educationList as |item|}}
+                        {{#if (and item.institution item.department)}}
+                            {{concat item.institution ', ' item.department}}
+                        {{else}}
+                            {{item.institution}}
+                        {{/if}}
+                    {{/each}}
+                {{else}}
+                    {{t 'osf-components.contributors.noAffiliations'}}
+                {{/if}}
```

Comment:
Get rid of this in favor of translated language for each case (no employment, no education) as @futa-ikeda suggested.
```suggestion
```